docs/user-manual/en/core.md

```diff
@@ -159,6 +159,19 @@ end asynchronous systems which is not possible using the standard JMS API. For
 more information on this advanced feature please see the section [Guarantees of
 sends and commits](send-guarantees.md).
 
+> **Identifying your session**
+>
+> Assigning IDs to your core sessions can save you from a bit of headache
+> when monitoring and debugging the cluster using the [management console](management-console.md).
+> ```java
+> ClientSession session;
+> // ...
+> session.addMetaData(ClientSession.JMS_SESSION_IDENTIFIER_PROPERTY, "jms-client-id");
```

Comment:
setClientID from JMS would do the same here.

Reply:
What if I don't use JMS though?